### DISCUSSION TITLE
[Feature request] Git ref `refs/release/latest` for the latest release

### DISCUSSION THREAD

**Discussion by lilyball** [community/community#123475] [Repositories]
### Select Topic Area

Product Feedback

### Body

The GitHub API has a way to ask for what the latest release is, but there's no way to get this information through the git protocol. As GitHub already has precedent for automatically creating special refs (the `refs/pull/` namespace), I'd really like to see it offer a `refs/release/latest` ref that points at the latest release (either at the annotated tag for the latest release, or at the commit). This would really help when using tooling that understands how to resolve Git refs but doesn't know about the GitHub API. For example, with a ref like this, I could use it with [Nix](https://nixos.org) flakes to keep my dependencies set to the latest release rather than `HEAD`.

**Comment by grandmas-favorite** [comment]
Hi @lilyball!

We appreciate your feedback and your valuable input. We want to be transparent about our current priorities. You can gain insights into the projects we’re currently focusing on for the upcoming year by checking our public roadmap.

While smaller fixes may not always be explicitly listed, they often get addressed when related work is in progress. We’d like to emphasize that as a valued GitHub user, your feedback in this Discussion space, along with appropriate tags and examples, plays a crucial role in shaping the development of GitHub. While we can’t guarantee immediate action on every piece of feedback, please rest assured that we are attentive to your suggestions and continuously strive to enhance the GitHub experience.
